weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/TraceUsageStats.tsx

```diff
@@ -91,6 +93,16 @@ export const getUsageFromCellParams = (params: {[key: string]: any}) => {
   return usage;
 };
 
+// This needs to updated eventually, to either include more possible keys or to be more dynamic
+// accounts for openai and anthropic usage objects (prompt_tokens, input_tokens)
+export const getInputTokens = (usage: UsageData) => {
+  return usage.prompt_tokens || usage.input_tokens || 0;
```

Comment:
I think you would want to use `??` and check for input_tokens first. Same comment for output calculation.